Widget/Dining/DiningAnalyticsHomeWidget.swift

```diff
@@ -94,6 +102,33 @@ extension DiningAnalyticsAuxiliaryStatistic {
     }
 }
 
+private func formatSwipes(statistic: DiningAnalyticsAuxiliaryStatistic, in balance: BalanceDetails<Int>, includeUnits: Bool = false) -> LocalizedStringKey {
+    switch statistic.getMetric(in: balance) {
+    case .amount(let amount):
+        if includeUnits {
+            // Auto-inflection is genuinely an amazing feature
+            return "^[\(amount) swipe](inflect: true)"
```

Comment:
This is wild I've never seen this

Reply:
Yeah lol, didn't realize it existed either